Find a functionally equivalent code:android.widget.TextView.getShadowColor:COMMENT
Method Modifier: public      
Comment:/**
 * @return the color of the shadow layer
 *
 * @see #setShadowLayer(float, float, float, int)
 *
 * @attr ref android.R.styleable#TextView_shadowColor
 */

@@ -1,3 +1,3 @@
 {
-    return mTextPaint.shadowColor;
+    return mShadowColor;
 }

